src/main/scala/com/ing/wbaa/rokku/sts/config/VaultSettings.scala

```diff
@@ -0,0 +1,21 @@
+package com.ing.wbaa.rokku.sts.config
+
+import akka.actor.{ ExtendedActorSystem, Extension, ExtensionId, ExtensionIdProvider }
+import com.typesafe.config.Config
+
+class VaultSettings(config: Config) extends Extension {
+
+  val vaultUrl: String = config.getString("vault.url")
+  val vaultPath: String = config.getString("vault.path")
+  val vaultToken: String = config.getString("vault.token")
+  val readTimeout: Int = config.getString("vault.read-timeout").toInt
```

Comment:
I think you can use getInt directly, like conf.getInt("foo.bar")

Reply:
fixed